ENIGMAsystem/SHELL/Graphics_Systems/General/GScolors.cpp

```diff
@@ -0,0 +1,111 @@
+/** Copyright (C) 2017 Faissal I. Bensefia
```

Comment:
Thanks again for doing this! That's a hundred lines of code you've spared this project from replicating in six places.

Reply:
Oh stop you're making me blush ^_^